controllers/gce/controller/utils_test.go

```diff
@@ -71,17 +71,12 @@ func TestInstancesAddedToZones(t *testing.T) {
 
 	// Create 2 igs, one per zone.
 	testIG := "test-ig"
-	testPort := int64(3001)
-	lbc.CloudClusterManager.instancePool.AddInstanceGroup(testIG, testPort)
+	lbc.CloudClusterManager.instancePool.AddInstanceGroup(testIG, []int64{int64(3001)})
 
 	// node pool syncs kube-nodes, this will add them to both igs.
 	lbc.CloudClusterManager.instancePool.Sync([]string{"n1", "n2", "n3"})
 	gotZonesToNode := cm.fakeIGs.GetInstancesByZone()
 
-	if cm.fakeIGs.Ports[0] != testPort {
```

Comment:
Removed this since we already have a dedicated test for testing ports on instance groups. No need to duplicate the check here.